invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/LinksSearchResultContainer.js

```diff
@@ -118,6 +120,26 @@ export class LinksSearchResultContainer extends Component {
     }
   };
 
+  generateDropdownOptions = () => {
+    const { record } = this.props;
+
+    // "can view" option is disabled for drafts
+    const dropdownOptionsCopy = _cloneDeep(dropdownOptions);
+    if (record?.is_draft || record?.is_draft === null) {
+      const viewOption = dropdownOptionsCopy.find((item) => item.key === "view");
+      viewOption.text = i18next.t(
+        "Can view (view access link can be created only after the record is published)"
+      );
```

Comment:
Not ideal, but shall we keep the text coming from the backend, and append a short msg?
```suggestion
      viewOption.text += i18next.t(
        " (unavailable: record not published yet)"
      );
```